### PR TITLE
Made IbisTestEnv imported locally in fs test

### PR DESCRIPTION
**Pull request by matthewmturner**
For #2467

### CODE REVIEW

**Comment by datapythonista** [comment]
I think isort introduced some changes that are breaking the CI, can you have a look please? May be you need to update to the latest version (we should probably pin it, isort is quite annoying).

**Review by jreback — APPROVED** [review]
lgtm merge on green.

File: setup.cfg [hunk 1]
```diff
@@ -16,7 +16,7 @@ inherit = false
 convention = numpy
 
 [isort]
-known_third_party = asv,click,clickhouse_driver,dateutil,google,graphviz,impala,kudu,mock,multipledispatch,numpy,pandas,pkg_resources,plumbum,psycopg2,pyarrow,pydata_google_auth,pygit2,pymapd,pymysql,pyspark,pytest,pytz,regex,requests,setuptools,sphinx_rtd_theme,sqlalchemy,thrift,toolz
+known_third_party = click,clickhouse_driver,dateutil,google,graphviz,impala,kudu,mock,multipledispatch,numpy,pandas,pkg_resources,plumbum,psycopg2,pyarrow,pydata_google_auth,pygit2,pymapd,pymysql,pyspark,pytest,pytz,regex,requests,setuptools,sphinx_rtd_theme,sqlalchemy,thrift,toolz
```
Comment (jreback) [inline setup.cfg:19]: we changed the commit hook so i am not sure this would change anymore. can you merge master and see?

Comment (matthewmturner) [reply setup.cfg]: I merged and ran precommit on all files but it didnt add asv back to the isort `known_third_party`.

So i tried manually adding asv and running precommit again and it kept it.

Is this expected?

Comment (jreback) [reply setup.cfg]: hmm weird, ok looks fine to me

**Comment by jreback** [comment]
lgtm ping on green.

**Comment by jreback** [comment]
thanks @matthewmturner

**Pull request merged**